docs/modules/ROOT/pages/utilities.adoc

```diff
@@ -1,320 +1,289 @@
 = Utilities
 
-The following documentation provides context, reasoning, and examples for methods and constants found in `tests/utils.py`.
+The following documentation provides reasoning and examples for functions and constants found in `openzeppelin::utils`
+and `openzeppelin::tests::utils`.
 
 CAUTION: Expect this module to evolve (as it has already done).
 
-== Table of Contents
+== Core utilities
 
-* <<constants,Constants>>
-* <<strings,Strings>>
- ** <<str_to_felt,`str_to_felt`>>
- ** <<felt_to_str,`felt_to_str`>>
-* <<uint256,Uint256>>
- ** <<uint,`uint`>>
- ** <<to_uint,`to_uint`>>
- ** <<from_uint,`from_uint`>>
- ** <<add_uint,`add_uint`>>
- ** <<sub_uint,`sub_uint`>>
-* <<assertions,Assertions>>
- ** <<assert_revert,`assert_revert`>>
- ** <<assert_revert_entry_point,`assert_revert_entry_point`>>
- ** <<assert_event_emitted,`assert_events_emitted`>>
-* <<memoization,Memoization>>
- ** <<get_contract_class,`get_contract_class`>>
- ** <<cached_contract,`cached_contract`>>
-* <<state,State>>
-* <<account,Account>>
-* <<mocksigner,MockSigner>>
+[.contract]
+[[utils]]
+=== `++utils++`
 
-== Constants
+```javascript
+use openzeppelin::utils;
+```
 
-To ease the readability of Cairo contracts, this project includes reusable https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.6.1/src/openzeppelin/utils/constants/library.cairo[constant variables] like `UINT8_MAX`, or EIP165 interface IDs such as `IERC165_ID` or `IERC721_ID`.
-For more information on how interface ids are calculated, see the xref:introspection.adoc#interface_calculations[ERC165 documentation].
+Module containing core utilities of the library.
 
-== Strings
+[.contract-index]
+.Members
+--
+.Functions
+* xref:#utils-try_selector_with_fallback[`++try_selector_with_fallback(target, selector1, selector2, args)++`]
```

Comment:
Not closing the thread to get @martriay opinion

Reply:
I agree, the only issue I have with first_selector is that it looks broken in UI, because is too large, but I guess we can live with it:

<img width="741" alt="Screenshot 2023-11-27 at 12 44 48" src="https://github.com/OpenZeppelin/cairo-contracts/assets/26070224/3e167a6b-c159-4f83-bf3e-3df6a56b0f71">

vs 

<img width="740" alt="Screenshot 2023-11-27 at 12 45 06" src="https://github.com/OpenZeppelin/cairo-contracts/assets/26070224/d55be90b-edfc-428b-b3ba-23d30f00c0ed">

Reply:
Maybe this works though?

<img width="740" alt="Screenshot 2023-11-27 at 12 46 34" src="https://github.com/OpenZeppelin/cairo-contracts/assets/26070224/401056dc-7758-4e5c-b8eb-3e0d3b234213">

Reply:
I forgot to mention that i personally don't like first and second without the selector, because is not clear enough about what the param is referring to IMO.

Reply:
cc @martriay. tagging because the PR has two approves, and I was about to merge it, but I realized this thread was still open